### PR TITLE
add context for mirror failures

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Review by poornas — APPROVED** [review]
LGTM

File: cmd/mirror-main.go [hunk 1]
```diff
@@ -384,7 +384,12 @@ func (mj *mirrorJob) monitorMirrorStatus() (errDuringMirror bool) {
 					fmt.Sprintf("Failed to remove `%s`.", sURLs.TargetContent.URL.String()))
 				errDuringMirror = true
 			default:
-				errorIf(sURLs.Error.Trace(), "Failed to perform mirroring.")
+				if sURLs.ErrorCond == differInUnknown {
+					errorIf(sURLs.Error.Trace(), "Failed to perform mirroring")
+				} else {
+					errorIf(sURLs.Error.Trace(),
+						"Failed to perform mirroring, with error condition (%s)", sURLs.ErrorCond)
```
Comment (vadmeste) [inline cmd/mirror-main.go:391]: I am not exactly sure what do you mean by error condition here? Could you explain ?

Comment (harshavardhana) [reply cmd/mirror-main.go:391]: The type of condition why we couldn't mirror @vadmeste

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**